### ISSUE TITLE
Support of Object Type

### ISSUE THREAD

**Issue by gtsluk** [ayushnawani/react-native-alphabetflatlist#2]
Can support Object rather than single value ?
For Example 
Object.label used for display on the flatlist
Object.id is the key

**Comment by ayushnawani** [comment]
Hey @gtsluk,
I have added a new props matchFieldName using which you can define the name of the field  to  match with letters. 

Latest changes are pushed into [**dev**](https://github.com/ayushnawani/react-native-alphabetflatlist/commits/dev) branch, You can try dev branch code and  let me know whether it helps you or not,
 
Example: For data like 

```
const testData =  [{
    label: 'kabel1',
    id: 1,
  },
  {
    label: 'label2',
    id: 2,
  },
  {
    label: 'label3',
    id: 3,
  }]
```

Pass matchFieldName in AlphabetFlatList like this
```
        <AlphabetFlatList
          renderItem={this.renderItem}
          ListHeaderComponent={this.renderHeader}
          ListFooterComponent={this.renderFooter}
          data={testData}
          getItemLayout={this.getItemLayout}
          keyExtractor={(item, index) => {
            return item.id;
          }}
          matchFieldName="label" // define the field name to match with letter
        />
```

**Comment by gtsluk** [comment]
it works. Thanks

**Comment by ayushnawani** [comment]
Great. I have released new version v1.0.4 with these changes.
